src/index.html

```diff
@@ -15,34 +15,16 @@ <h1 id="title">Leaderboard</h1>
       <div id="scoreList">
         <div id="listHeader">
           <h3>Recent scores</h3>
-          <button>Refresh</button>
+          <button id="refresh">Refresh</button>
         </div>
-        <div class="scoreItem">
-          <h2 class="scorer">C. Ronaldo</h2>
-          <p class="score">89</p>
-        </div>
-        <div class="scoreItem">
-          <h2 class="scorer">Mbappe</h2>
-          <p class="score">92</p>
-        </div>
-        <div class="scoreItem">
-          <h2 class="scorer">Messi</h2>
-          <p class="score">96</p>
-        </div>
-        <div class="scoreItem">
-          <h2 class="scorer">L.Diaz</h2>
-          <p class="score">75</p>
-        </div>
-        <div class="scoreItem">
-          <h2 class="scorer">M. Salah</h2>
-          <p class="score">88</p>
+        <div id="scoreItems">
         </div>
       </div>
       <div id="scoreForm">
         <h3>Add your score</h3>
-        <form action="#" id="addScore">
-          <input type="text" placeholder="Your name" name="name" class="scoreField">
-          <input type="text" placeholder="Your score" name="score" class="scoreField">
+        <form id="addScore">
+          <input type="text" placeholder="Your name" name="scorer" class="scoreField" id="scorer">
+          <input type="text" placeholder="Your score" name="score" class="scoreField" id="score">
```

Comment:
[Optional] I suggest you validate the name and score fields to ensure a user cannot submit blank entries 👍

Reply:
1. Already done in index.js